src/Hazelcast.Net.Tests/Remote/ClientBaseCollectionTest.cs

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Hazelcast.DistributedObjects;
+using Hazelcast.Testing;
+using NUnit.Framework;
+
+namespace Hazelcast.Tests.Remote
+{
+    public abstract class ClientBaseCollectionTest : SingleMemberClientRemoteTestBase
```

Comment:
This is a base, abstract class → should be something like `ClientCollectionTestBase`.

Reply:
ok